integration/create_test.go

```diff
@@ -30,7 +30,7 @@ import (
 )
 
 func TestCreateStress(t *testing.T) {
-	// TODO rewrite using teststress.Stress
+	// It should be rewritten to use teststress.Stress.
```

Comment:
Don't we want to make issues for those?

Reply:
No, that just needs to be done when we change that test the next time